### PR TITLE
Update faq.md about enabling detailed HTTP logging

### LINKED ISSUE #4039: Hard to follow network-traffic troubleshooting instructions
**Environment**:

- *Jib version:* 3.3.2
- *Build tool:* Gradle 8.1
- *OS:* Ubuntu 22.04.2 LTS

**Description of the issue**:
The Jib troubleshooting involves running Gradle with `--no-daemon` which is challenging to do. 
See  https://github.com/gradle/gradle/issues/16110 

https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#how-can-i-examine-network-traffic

**Expected behavior**:
Troubleshooting will not involve running with `--no-daemon` .
Actually, after some additional testing, I discovered that adding `--no-daemon`  is not required, but one must use `--info` flag. Maybe the documentation needs to be updated?

### PR DESCRIPTION
**Pull request by chanseokoh**
Fixes #4039.

### CODE REVIEW

**Review by mpeddada1 — APPROVED** [review]
Thank you so much @chanseokoh!

**Comment by chanseokoh** [comment]
Please merge!
